label_studio/data_manager/actions/basic.py

```diff
@@ -88,7 +88,7 @@ def delete_tasks_annotations(project, queryset, **kwargs):
     annotations.delete()
     emit_webhooks_for_instance(project.organization, project, WebhookAction.ANNOTATIONS_DELETED, annotations_ids)
     start_job_async_or_sync(bulk_update_stats_project_tasks, queryset.filter(is_labeled=True))
-    start_job_async_or_sync(project.update_tasks_counters, queryset)
+    start_job_async_or_sync(project.update_tasks_counters, list(queryset))
```

Comment:
what if we have 1m tasks here? they will be downloaded fully, but we can use only ids here

Reply:
Fixed